python/mlx/nn/losses.py

```diff
@@ -2,7 +2,46 @@
 
 import mlx.core as mx
 
+def cross_entropy(logits: mx.array, targets: mx.array, axis: int = -1, reduction: str = 'mean'):
```

Comment:
could you default `reduction` to `'none'` for now? I know it's different than PyTorch but we have downstream code which depends on this function that will otherwise break.